rhine-terminal/src/FRP/Rhine/Terminal.hs

```diff
@@ -0,0 +1,103 @@
+{- | Wrapper to write @terminal@ applications in Rhine, using concurrency.
+-}
+
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE UndecidableInstances #-}
```

Comment:
Oh no, why is this needed?

Reply:
Must have been previous flailing, removing it.